project/SparkRedshiftBuild.scala

```diff
@@ -55,6 +55,8 @@ object SparkRedshiftBuild extends Build {
       resolvers +=
         "Spark 1.5.0 RC3 Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1143",
       libraryDependencies ++= Seq(
+        // The SLF4J API is provided by Spark:
+        "org.slf4j" % "slf4j-api" % "1.7.5" % "provided",
```

Comment:
I noticed that `spark-csv` also marks it as provided, but I think you're right: this should be included because it doesn't have any dependencies of its own. I'll update to do this now, since it would be better to set a good example here.